docs/installation.md

```diff
@@ -50,6 +50,21 @@ To update ty, use `uv tool upgrade`:
 uv tool upgrade ty
 ```
 
+### Using mise
```

Comment:
If we include this (which I think will require consensus from other maintainers too), I think it should go below all the official distribution methods instead of here. The title should also probably be consistent? e.g., "Installing with mise"?